tests/src/c_source/ut_test_kvp_profile.c

```diff
@@ -143,8 +143,22 @@ void test_ut_kvp_profile_open( void )
 {
     UT_LOG_STEP( "test_ut_kvp_profile_open - start" );
     ut_kvp_profile_open( KVP_VALID_TEST_ASSERT_YAML_FILE );
-    ut_kvp_profile_open( KVP_VALID_TEST_ASSERT_YAML_FILE );
-    ut_kvp_profile_open( KVP_VALID_TEST_ASSERT_YAML_FILE );
+    UT_LOG_STEP( "ut_kvp_profile_open - %s", KVP_VALID_TEST_ASSERT_YAML_FILE);
+
+    ut_kvp_instance_t *pInstance;
+
+    pInstance = ut_kvp_profile_getInstance();
+    char* kvpdata = ut_kvp_getData(pInstance);
+
+    if(kvpdata != NULL)
+    {
+        // Print the emitted KVP string
+        printf("%s\n", kvpdata);
```

Comment:
how does that test whether it worked or not.. don't you need to ccheck a known value in the 2nd include to prove the 2nd include worked?

Reply:
sure, I didn't do that way as I thought printing the merged yaml will be easier . I can add that test as well.